### PR TITLE
WIP: #566 duplicate fields

### PR DESCRIPTION
**Pull request by noumar**
#566

### CODE REVIEW

File: invenio/b2share/modules/b2deposit/b2share_model/model.py [hunk 1]
```diff
@@ -95,13 +96,11 @@ def __init__(self):
...
         self.field_args['publisher'] = {
-            'hidden': True,
-            'value': self.publisher_default
-            # 'description':
-            # 'Here should be stored the site that will host the BE2Share ' +\
-            # 'container, so that in case of access problems, people can ' +\
-            # 'be contacted. This element can be created automatically ' +\
-            # 'dependent on the centre.'
+            'value': self.publisher_default,
+            'description': 'Here should be stored the site that will host the B2SHARE '
```
Comment (emanueldima) [inline invenio/b2share/modules/b2deposit/b2share_model/model.py]: This description must be changed

File: invenio/b2share/modules/b2deposit/b2share_model/model.py [hunk 2]
```diff
@@ -140,10 +139,8 @@ def __init__(self):
                            'were relevant in the creation of the resource.'
         }
         self.field_args['language'] = {
-            'hidden': True,
             'value': self.language_default,
-            # 'description':
-            # 'The name of the language the document is written in.'
+            'description': 'The name of the language the document is written in.'
```
Comment (emanueldima) [inline invenio/b2share/modules/b2deposit/b2share_model/model.py]: The Linguistics domain also has a language_code field. We must avoid having 2 different fields for the same thing, especially if they can be filled in by the user with contradictory information.

**Comment by emanueldima** [comment]
I agree with the patch in general, but see the line comments.

File: invenio/b2share/modules/b2deposit/b2share_model/model.py (hunk 1 above)
Comment (noumar) [inline invenio/b2share/modules/b2deposit/b2share_model/model.py]: Agree, but what "should" the field represent? The name, description, common use and our use are not consistent. We assign it automatically a "service instance url" but it's unclear to me why so, and e.g. an article could already have been published in a magazine to which the uploader wants it to refer.

@cjhak can you maybe give some input on this? Specifically use and description.

File: invenio/b2share/modules/b2deposit/b2share_model/model.py [hunk 3]
```diff
@@ -95,13 +96,11 @@ def __init__(self):
...
         }
         self.field_args['publisher'] = {
-            'hidden': True,
-            'value': self.publisher_default
-            # 'description':
-            # 'Here should be stored the site that will host the BE2Share ' +\
-            # 'container, so that in case of access problems, people can ' +\
-            # 'be contacted. This element can be created automatically ' +\
-            # 'dependent on the centre.'
+            'value': self.publisher_default,
```
Comment (noumar) [inline invenio/b2share/modules/b2deposit/b2share_model/model.py]: @emanueldima do you know how to handle the value so it isn't stored as `None` in the DB? It only occurs if the user empties the prefilled field.

File: invenio/b2share/modules/b2deposit/b2share_model/model.py (hunk 2 above)
Comment (noumar) [inline invenio/b2share/modules/b2deposit/b2share_model/model.py]: Agree, but this doesn't seem to be used, at least not visible, anywhere on the record page. So what's been the purpose of the field?

**Comment by emanueldima** [comment]
I'll merge this one and try to fix the problems later as it's important to have the rda domain integrated fast

**Pull request merged**